data/campaigns/Winds_of_Fate/about.cfg

```diff
@@ -18,7 +18,7 @@
     [/entry]
     [entry]
         name="inferno8"
-        comment="storm daemon unit sprite and attack icons from Era of Magic"
+        comment="storm wisp unit sprite and attack icons from Era of Magic"
```

Comment:
I kinda forgot to check this when we first merged the campaign, but did you add licensing for these images? I think EoMa uses CC BY-SA but not entirely sure.

Reply:
This sprite in particular is GNU GPLv2 from the first(?) release of Era of Magic back in the 1.10 days. I will have to check but I think the others are old enough to be from that same time period where all art was licensed GPL along with the code.

Reply:
Can you please check? I would like to make sure we respect inferno8's wishes regarding licensing on his own work.

Reply:
I did, it is GNU GPLv2. You can confirm by going to https://addons.wesnoth.org/1.10/ and downloading Era of Magic from there. The relevant files are named `COPYING` (the GPL license itself) and `CREDITS` (lists inferno8 as the sole artist). The sprite in question was replaced by another and removed from later versions of EoMa so we have to use the licensing provided in this older version found on the BfW 1.10 addons server archive.

Also I asked inferno8 for his explicit permission to use the sprite in WoF and [he kindly gave it back here.](https://github.com/wesnoth/wesnoth/pull/6875#issuecomment-1186109407)